vpr/src/place/initial_placement.cpp

```diff
@@ -366,6 +553,8 @@ static void place_macro(int macros_max_num_tries, t_pl_macro pl_macro, enum e_pa
             break;
         }
     }
+
```

Comment:
Delete the creation of the string that is never printed (looks like leftover debugging code)